modules/wipesBidAdapter.js

```diff
@@ -0,0 +1,74 @@
+import * as utils from '../src/utils.js';
+import {config} from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'wipes';
+const ALIAS_BIDDER_CODE = ['wi'];
+const SUPPORTED_MEDIA_TYPES = [BANNER]
+const ENDPOINT_URL = 'https://adn-srv.reckoner-api.com/v1/prebid';
+
+function isBidRequestValid(bid) {
+  switch (true) {
+    case !!(bid.params.asid):
+      break;
+    default:
+      utils.logWarn(`isBidRequestValid Error. ${bid.params}, please check your implementation.`);
+      return false;
+  }
+  return true;
+}
+
+function buildRequests(validBidRequests, bidderRequest) {
+  return validBidRequests.map(bidRequest => {
+    const params = bidRequest.params;
+    const asid = params.asid;
+    return {
+      method: 'GET',
+      url: ENDPOINT_URL,
+      data: {
+        asid: asid,
+        mediaTypes: {
+          video: {
+            context: 'outstream'
+          }
+        },
+      }
+    }
+  });
+}
+
+function interpretResponse(serverResponse, bidRequest) {
+  const bidResponses = [];
+  const response = serverResponse.body;
+  const cpm = response.cpm * 1000 || 0;
+  if (cpm !== 0) {
+    const netRevenue = (response.netRevenue === undefined) ? true : response.netRevenue;
+    const bidResponse = {
+      requestId: response.uuid,
+      cpm: cpm,
+      width: response.width,
+      height: response.height,
+      creativeId: response.videoCreativeId || 0,
+      dealId: response.dealId,
+      currency: 'JPY',
+      netRevenue: netRevenue,
+      ttl: config.getConfig('_bidderTimeout'),
+      referrer: bidRequest.data.r || '',
+      mediaType: BANNER,
+      ad: response.adTag,
```

Comment:
In the network response I see `ad_tag`, not `adTag` -- please fix it 

Also, check all other camelCase variables: netRevenue, videoCreativeId, dealId

Reply:
@musikele 
I'm sorry, I updated again. It's ready to be reviewed